assets/aml-benchmark/requirements.txt

```diff
@@ -0,0 +1,10 @@
+pytest
```

Comment:
nit: do we need requirements.txt as well?

Reply:
will remove this as this is no longer required.